kafka-connect-neo4j/src/main/kotlin/streams/kafka/connect/source/Neo4jSourceConnector.kt

```diff
@@ -27,7 +25,9 @@ class Neo4jSourceConnector: SourceConnector() {
         config = Neo4jSourceConnectorConfig(settings)
     }
 
-    override fun stop() {}
+    override fun stop() {
+        ConfigurationMigrator(settings).migrate()
```

Comment:
Should we also include stored offset value (by dumping the configuration by setting `start-from` to `user defined` and `start-from.value` to the last stored offset) with the source connector, wdyt?

Reply:
Is this a configuration option that didn't exist in the original connector, but does in 5.1? Would that be handled by defaults?